js/languages/en-US.json

```diff
@@ -71,12 +71,13 @@
   "Welcome": "Welcome",
   "CreateStore": "Become a Store",
   "GoToMyPage": "My Page",
-  "ShowOnlyFollowerListings": "Filtered Listings",
+  "Random": "Random",
+  "ShowOnlyFollowerListings": "From Friends",
```

Comment:
I've been struggling with the terminology here. I switched it to Personalized which seems to work a lot better. The tooltip explains what that term means then when you hover.